client/src/components/Form.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import WeatherCard from "./WeatherCard";
+import "./Form.css";
+import { getWeatherForLocation } from "../utils";
+import InputRow from "./InputRow";
+
+const Form = ({ setUserFavorites }) => {
```

Comment:
I think avoiding naming components similarly to existing HTML tags isn't great. `Form` sounds generic, but this component has a specific purpose.